docs/user_manual/model-validation.md

```diff
@@ -0,0 +1,133 @@
+<!--
+SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# Model validation
+
+power-grid-model is validated using multiple cases present in [/tests/data](../../tests/data) folder.
```

Comment:
relative links to data folder will not work. maybe just use absolute http url

Reply:
Fixed them